fcrepo4/fcrepo4
# Add integration test for repository rebuild functionality

Resolves: https://jira.lyrasis.org/browse/FCREPO-3210

# What does this Pull Request do?
This pull-request adds one new integration test for verifying the repository rebuild functionality.
In adding this test, the OCFLConstants.java class was updated to allow for runtime update of the OCFL storage root, staging, and work directories.
Most of the files in this pull-request are new test fixtures.

# How should this be tested?
The application should work exactly as before (one-click, etc).
A successful build is a successful test.

# Interested parties
Tag @pwinckles / @whikloj
